Rubberduck.Resources/Inspections/InspectionInfo.resx

```diff
@@ -217,7 +217,7 @@ If the parameter can be null, ignore this inspection result; passing a null valu
   <data name="SelfAssignedDeclarationInspection" xml:space="preserve">
     <value>An auto-instantiated object variable declaration at procedure scope changes how nulling the reference works, which can lead to unexpected behavior.</value>
   </data>
-  <data name="FunctionReturnValueNotUsedInspection" xml:space="preserve">
+  <data name="FunctionReturnValueNeverUsedInspection" xml:space="preserve">
     <value>A member is written as a function, but used as a procedure. Unless the function is recursive, consider converting the 'Function' into a 'Sub'. If the function is recursive, none of its external callers are using the returned value.</value>
```

Comment:
IIUC the updated code handles recursive functions - the `InspectionInfo` text should be amended accordingly, no?

Reply:
The inspection still does not report recursive functions; well, at least if the use their own return value.

Reply:
Hm, reading that again, it suggests that recursive functions are reported. However, the tests test that it is not the case.

Reply:
Should I change the description or actually implement the behaviour mentioned in the description, including appropriate tests?

Reply:
Nah the inspection should just do what it needs to do, the description likely mentions recursiveness because inspection info strings sometimes mention shortcomings of inspections' implementations - if the shortcoming is fixed, the mention of it serves no purpose anymore. Let's reword the description.